dmoj/cli.py

```diff
@@ -330,23 +331,19 @@ def execute(self):
         print highlight(file_diff, guess_lexer(file_diff), terminal256())
```

Comment:
Should move [`_find_exe`](https://github.com/DMOJ/judge/blob/e922b9f7867958c53b2de97942bbc628d49629ff/dmoj/cptbox/sandbox.py#L22) into a helper in utils, then check if `_find_exe('git.exe' if os.name == 'nt' else 'git')` exists (function doesn't raise OSError), and if so use `subprocess.call(_find_exe(...), 'diff', 'file1', 'file2')` instead.